_playwright-tests/Integration/CanInstallUploadRepositryOnHost.spec.ts

```diff
@@ -0,0 +1,135 @@
+import path from 'path';
+import { test, expect, cleanupRepositories, cleanupTemplates, randomName } from 'test-utils';
+import { RHSMClient, refreshSubscriptionManager } from './helpers/rhsmClient';
+import { runCmd } from './helpers/helpers';
+import { navigateToRepositories, navigateToTemplates } from '../UI/helpers/navHelpers';
+import { closePopupsIfExist, getRowByNameOrUrl, retry } from '../UI/helpers/helpers';
+
+const uploadRepoName = 'Upload_Repo';
+test.describe('Install Upload Repositories on Host', () => {
```

Comment:
this test is installing content from a repository, so what do you think about renaming the test file to something like InstallUploadRepoContent?